OpenSim/Common/Object.h

```diff
@@ -808,11 +810,11 @@ class OSIMCOMMON_API Object
     void updateDefaultObjectsFromXMLNode();
     void updateDefaultObjectsXMLNode(SimTK::Xml::Element& aParent);
 
-    /** This is invoked at the start of print(). Derived classes can use this
-     * as an opportunity to issue warnings to users.
-     * Any exception thrown in this function is ignored, as exceptions would
-     * prevent the user from printing the object, which could be useful for
-     * debugging. */
+    /** This is invoked at the start of print() if second argument to print is true
+     * printing is allowed to proceed even if the resulting file is corrupt, otherwise
+     * printing is aborted.
```

Comment:
There is no second argument to `print()`.

Reply:
Nice catch @chrisdembia will update.

Reply:
Updatd.

Reply:
Should this be `_debugLevel is at least 1`?

Also, looks like this should be two sentences: `This is invoked at the start of print(). If _debugLevel is greater than 0 then ...`.